model/host/db.go

```diff
@@ -391,41 +387,45 @@ func NumHostsByTaskSpec(group, buildVariant, project, version string) (int, erro
 			},
 		},
 	)
-	hosts, err := Find(q)
+}
+
+// NumHostsByTaskSpec returns the number of running hosts that are running a task with
+// the given group, buildvariant, project, and version.
+func NumHostsByTaskSpec(group, buildVariant, project, version string) (int, error) {
+	if group == "" || buildVariant == "" || project == "" || version == "" {
```

Comment:
Should this logic be in `ByTaskSpec` so that it's not duplicated between `NumHostsByTaskSpec` and `MinTaskGrouporderRunninByTaskSpec`? (If not, a note that the error on 414 names the wrong function.)

Reply:
I think it shouldn't be since ByEtc in db files usually only construct queries. I'll fix 414.